### PR TITLE
Updated versions and code simplification

### PR DESCRIPTION
**Pull request by emdobrin**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by yangyansong-adbe — APPROVED** [review]

File: Sample-App/app/build.gradle [hunk 1]
```diff
@@ -39,36 +43,20 @@ dependencies {
     implementation 'com.google.android.gms:play-services-location:21.0.1'
     implementation 'com.google.code.gson:gson:2.8.6'
 
-    implementation 'com.adobe.marketing.mobile:core:2.0.0-SNAPSHOT'
+    implementation 'com.adobe.marketing.mobile:core:2.0.0'
```
Comment (praveek) [inline Sample-App/app/build.gradle:46]: Can you update the core extensions to the latest available version (2.0.1)?

File: Sample-App/app/build.gradle [hunk 2]
```diff
@@ -39,36 +43,20 @@ dependencies {
...
+    implementation'com.adobe.marketing.mobile:lifecycle:2.0.0'
+    implementation 'com.adobe.marketing.mobile:identity:2.0.0'
+    implementation 'com.adobe.marketing.mobile:signal:2.0.0'
+    implementation 'com.adobe.marketing.mobile:assurance:2.0.0'
+    implementation 'com.adobe.marketing.mobile:userprofile:2.0.0'
+    implementation 'com.adobe.marketing.mobile:edge:2.0.0'
+    implementation 'com.adobe.marketing.mobile:edgeconsent:2.0.0'
+    implementation 'com.adobe.marketing.mobile:edgeidentity:2.0.0'
+
+    implementation 'com.adobe.marketing.mobile:messaging:1.+'
```
Comment (praveek) [inline Sample-App/app/build.gradle:56]: Can you update messaging to 2.0.1?

File: Sample-App/app/build.gradle [hunk 3]
```diff
@@ -39,36 +43,20 @@ dependencies {
...
     implementation('com.adobe.marketing.mobile:analytics:2.0.0-SNAPSHOT'){
```
Comment (praveek) [inline Sample-App/app/build.gradle:57]: Can you also update analytics to 2.0.0?

**Comment by praveek** [comment]
Can you switch the dependencies to dynamic version? Can you also add a warning about dynamic dependency and a link to this page https://developer-stage.adobe.com/client-sdks/documentation/manage-gradle-dependencies/

File: Sample-App/app/build.gradle [hunk 4]
```diff
@@ -39,36 +43,18 @@ dependencies {
...
-        transitive = false
-    }
-    implementation('com.adobe.marketing.mobile:edgeidentity:2.0.0-SNAPSHOT'){
-        transitive = false
-    }
-
-    implementation 'com.adobe.marketing.mobile:messaging:1.+'
+    // Using dynamic dependency versions is not recommended for production apps. For more information, please read https://developer.adobe.com/client-sdks/documentation/manage-gradle-dependencies
+    implementation 'com.adobe.marketing.mobile:core:2.+'
+    implementation'com.adobe.marketing.mobile:lifecycle:2.+'
```
Comment (praveek) [inline Sample-App/app/build.gradle:48]: nit: space here

**Review by praveek — APPROVED** [review]

**Pull request merged**